docs/variants/msi_z690/montage.md

```diff
@@ -0,0 +1,123 @@
+# Dasharo compatible with MSI PRO Z690-A DDR4 - montage
+
+## Intro
+
+This document describes the montage procedure of the MSI PRO Z690-A DDR4 with
```

Comment:
Let's name this procedure not montage but assembly

Reply:
renamed